client/src/utils/botExecution.jsx

```diff
@@ -0,0 +1,8 @@
+// is a dummy function
+const isBotExecutable = () => {
+    if (Math.random() > 0.49) {
```

Comment:
Pretty sure, this is not a good thing to do... What is its functionality? Why is a random number needed for it that seems to be a 50/50 chance generator